python/lsst/meas/astrom/matchPessimisticB.py

```diff
@@ -402,8 +402,12 @@ def _doMatch(self, refCat, sourceCat, wcs, refFluxField, numUsableSources,
         # the consensus mode of the matcher. If not default to Optimistic
         # pattern matcher behavior.
         numConsensus = self.config.numPatternConsensus
-        if ref_array.shape[0] < self.config.numBrightStars or \
-           src_array.shape[0] < self.config.numBrightStars:
+        minRefAndSrcObjectsForConsensus = \
+            self.config.numBrightStars + self.config.numPointsForShapeAttempt
+        if ref_array.shape[0] < self.config.numBrightStars + \
+           self.config.numPointsForShapeAttempt or \
+           src_array.shape[0] < self.config.numBrightStars + \
+           self.config.numPointsForShapeAttempt:
```

Comment:
Thanks for defining `minRefAndSrcObjectsForConsensus` but then please use it in the test. (flake8 should have warned you that the variable is unused?) Also, consider shortening the name to `minObjectsForConsensus`.